make-stdlib

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Build the Silver standard libraries.
+# Note that this does not rebuild the silver:core and the runtime.
+
+set -e
+
+export SILVER_HOME=$(pwd)
+JVM_ARGS="-Xss20M -Xmx1G -jar ../jars/silver.compiler.composed.Default.jar --no-stdlib --relative-jar $@"
```

Comment:
shellcheck complains,

```
In make-stdlib line 9:
JVM_ARGS="-Xss20M -Xmx1G -jar ../jars/silver.compiler.composed.Default.jar --no-stdlib --relative-jar $@"
         ^-- SC2124 (warning): Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```

it has a bunch of other warnings too, but i _think_ they're all "false positives for our purposes (e.g. idt silver can make a jar file that starts with a `-`)" -- they're all easy to fix, though

Reply:
lots of scripts are doing this; i think two reasonable things to do might be

- pass `"$@"` (note quotes) after `$JVM_ARGS` (note no quotes) to `java`
- make `JVM_ARGS` an array and do what shellcheck says